bin/cli.js

```diff
@@ -30,6 +30,8 @@ program
                 process.exitCode = 1;
             }
         } else {
+            console.log("Off-Roku BrightScript interpreter [Version " + packageJson.version + "]");
```

Comment:
Great addition!  Consider using a [template literal](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals#Expression_interpolation) instead of concatenating strings, e.g.

```suggestion
            console.log(`Off-Roku BrightScript interpreter [Version ${packageJson.version}`);
```